microcosm_fastapi/conventions/schemas.py

```diff
@@ -37,6 +37,10 @@ class _SearchSchema(BaseModel):
         links: Optional[LinksSchema] = Field(alias="_links")
         count: int
         items: List[item_class]
+        offset: Optional[int]
+        limit: Optional[int]
```

Comment:
I had a test case on the POC service that required offset and limit be part of the response